internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1106,10 +1100,16 @@ func configReplicaSetAWSProvider(projectID, name string, diskSizeGB, nodeCountEl
 func checkReplicaSetAWSProvider(projectID, name string, diskSizeGB, nodeCountElectable int, checkDiskSizeGBInnerLevel, checkExternalID bool) resource.TestCheckFunc {
 	additionalChecks := []resource.TestCheckFunc{
 		resource.TestCheckResourceAttr(resourceName, "retain_backups_enabled", "true"),
-		resource.TestCheckResourceAttrWith(resourceName, "replication_specs.0.region_configs.0.electable_specs.0.disk_iops", acc.IntGreatThan(0)),
+	}
+	if config.AdvancedClusterV2Schema() {
+		// Manual test fix to remove electable_specs.0 for TPF
```

Comment:
@lantoli missing conversion to v2 schema here. Any reason why we cannot expose `acc.attrNameToSchemaV2`?

Reply:
@EspenAlbert why do you need to do that?    `checkAggr` that is called at the end of this func calls `ConvertToTPFAttrsMap` and `ConvertToTPFAttrsSet` ?

Reply:
oh, I see, it's not done for extra params in checkAggr.  Yes, it's fine to make that func public, good catch

Reply:
Exposed in 0bba884833c3e1a91e3f88e4f0853c0eea936251